code/modules/spells/targeted/ethereal_jaunt.dm

```diff
@@ -68,7 +68,7 @@
 		SM.silence_spells(0)
 	target.flags &= ~INVULNERABLE
 	target.incorporeal_move = previncorp
-	target.alpha = prevalpha
+	target.alphas["etheral_jaunt"] = 255
```

Comment:
probably better to just remove it from the list than this, same with chameleon when it gets deactivated as a gene